### PR TITLE
[Test] Automate `Workspace stopping by idle timeout` test-case

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
We need to automate `Workspace stopping by idle timeout` test-case. To set secondsOfInactivityBeforeIdling property in CheClaster(patch) use KubernetesCommandLineToolsExecutor and ShellExecutor classes.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-4836

### How to test this PR?
Test scenario:
1. Set spec.devEnvironments.secondsOfInactivityBeforeIdling:180 in CheCluster
2. Start workspace using VS Code Editor
3. Wait 3+ minutes (it should be 60 minutes by default).
4. Make sure test workspace status is stopped in Dashboard.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by nallikaea — APPROVED** [review]

File: tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+		await workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
+	});
+
+	test('Wait workspace readiness', async function (): Promise<void> {
+		registerRunningWorkspace(WorkspaceHandlingTests.getWorkspaceName());
+		await projectAndFileTests.waitWorkspaceReadinessForCheCodeEditor();
+		await projectAndFileTests.performTrustAuthorDialog();
+	});
+
+	test('Wait idle timeout dialog and click on "Return to Dashboard" button', async function (): Promise<void> {
```
Comment (dmytro-ndp) [inline tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts:81]: IMHO, it makes sense to leave a comment to the test that if requires "Setting spec.devEnvironments.secondsOfInactivityBeforeIdling:180 in CheCluster" to run successfully.

Comment (nallikaea) [reply tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts:81]: Maybe it makes sence to do it in suiteSetup() ? we have KubernetesCommandLineToolsExecutor and ShellExecutor classes

Comment (SkorikSergey) [reply tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts:81]: Ok. I will try to use it. Thanks.

Comment (nallikaea) [reply tests/e2e/specs/miscellaneous/WorkspaceIdleTimeout.spec.ts:81]: Don`t forget discard the changes in suiteTeardown() than =)

**Review by dmytro-ndp — APPROVED** [review]
LGTM

**Pull request merged**